crates/bevy_core_pipeline/src/tonemapping/tonemapping_shared.wgsl

```diff
@@ -1,5 +1,191 @@
 #define_import_path bevy_core_pipeline::tonemapping
 
+// -------------------------------------
+// ---- tonemapping from kajiya 0.1 ----
+// -------------------------------------
+// Rec. 709
+fn srgb_to_luminance(col: vec3<f32>) -> f32 {
+    return dot(vec3(0.2126, 0.7152, 0.0722), col);
+}
+
+fn rgb_to_ycbcr(col: vec3<f32>) -> vec3<f32> {
+	let m = mat3x3<f32>(0.2126, 0.7152, 0.0722, -0.1146,-0.3854, 0.5, 0.5,-0.4542,-0.0458);
```

Comment:
```suggestion
    let m = mat3x3<f32>(
        0.2126, 0.7152, 0.0722,
        -0.1146, -0.3854, 0.5,
        0.5, -0.4542, -0.0458
    );
```

Reply:
Could we add a comment that references from where these coefficients come?

Reply:
They came from @h3r2tic